Packages/ANCIENT-MEGALITHIC-MYSTERY-PACKAGE/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="atg-location" content="Cusco">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="../../css/main.css">
+	<link rel="stylesheet" type="text/css" href="../../css/global.css">
+	<!-- PACKAGE STYLES -->
+	<link rel="stylesheet" type="text/css" href="../../css/packages.css">
+	<title>ANCIENT MEGALITHIC MYSTERY PACKAGE</title>
+	<!-- Script for generating the data -->
+	<script src="../../js/Packages.js"></script>
+</head>
+<body>
+	<div class="first Cusco">
+		<div class=logo>
+			<img src="https://res.cloudinary.com/dlia7phdx/image/upload/v1564095577/ATG/atg-logo.png">
+		</div>
+		<div class="gradient">
+			<h3>ANCIENT MEGALITHIC MYSTERY PACKAGE</h3>
```

Comment:
same as above - if you want ALLCAPS, you can select it with CSS - also, i have to say it again, "package" is such a burnt word